libsolidity/AST.h

```diff
@@ -440,13 +440,22 @@ class FunctionDefinition: public Declaration, public VariableScope, public Docum
 class VariableDeclaration: public Declaration
 {
 public:
-	VariableDeclaration(SourceLocation const& _location, ASTPointer<TypeName> const& _type,
-							ASTPointer<ASTString> const& _name, ASTPointer<Expression> _value,
-							Visibility _visibility,
-							bool _isStateVar = false, bool _isIndexed = false):
+	VariableDeclaration(
+		SourceLocation const& _location,
+		ASTPointer<TypeName> const& _type,
+		ASTPointer<ASTString> const& _name,
+		ASTPointer<Expression> _value,
+		Visibility _visibility,
+		bool _isStateVar = false,
+		bool _isIndexed = false,
+		bool _isConstant = false):
 			Declaration(_location, _name, _visibility),
```

Comment:
I thik this block should only be indented one tab after `VariableDeclaration` now, while I would indent the block above one further tab not to align with this block here.